src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -427,10 +424,8 @@ export function fetchPastAppointments(startDate, endDate, selectedIndex) {
 export function fetchRequestDetails(id) {
   return async (dispatch, getState) => {
     const state = getState();
-    const { appointmentDetails, requestMessages } = state.appointments;
-    const pendingAppointments = selectPendingAppointments(state);
-    const request =
-      appointmentDetails[id] || pendingAppointments?.find(p => p.id === id);
```

Comment:
`selectPendingAppointments` filters our express care requests, so we'll just search all of `pending`.